R/deploy.R

```diff
@@ -580,7 +580,7 @@ poll_task <- function(task, wait = 1, callback = message) {
   code <- -1
   first <- 0
   while (!finished) {
-    task_data <- con$task(task$get_task()$task_id, wait = wait, first = first)
+    task_data <- con$task(task$task$task_id, wait = wait, first = first)
```

Comment:
You don't technically need this change right? `get_task()` just yields `self$task`. (Not opposed to ripping out the unnecessary getters, but that seems better done in a separate refactor PR where you did them all.)

Reply:
Correct, I don't technically need it; happy to reverb if you think it's *bad* to change it here.

Reply:
I agree, for things like this which are easily greppable, no need to do it piecemeal.